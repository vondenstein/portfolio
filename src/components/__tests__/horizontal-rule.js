import React from "react"
import renderer from "react-test-renderer"

import HorizontalRule from "../horizontal-rule"

describe("HorizontalRule", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<HorizontalRule />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
