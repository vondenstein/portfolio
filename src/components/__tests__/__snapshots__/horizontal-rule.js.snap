// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HorizontalRule renders correctly 1`] = `
<div
  style={
    Object {
      "borderBottom": "1px solid grey",
      "margin": "140px auto 50px",
      "position": "relative",
    }
  }
/>
`;
